software-templates/techdocs-template/template.yaml

```diff
@@ -131,6 +131,58 @@ spec:
                         - "Other platform that isn't listed"
                         - "Not sure"
                         - "Not applicable"
+        -   title: Select a license for your repository
+            required:
+                - licenseType
+            properties:
+                licenseType:
+                    title: The repository license for your documentation
+                    type: string
+                    description: |
+                        Select a license for your technical documentation
+
+                        [Visit the BC Developer Guide](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/license-your-github-repository/#preferred-licenses)
+                        to read more about licensing your repository
+                    default: CreativeCommons
+                    enum:
+                        - None
+                        - CreativeCommons
+                        - Apache2
+                    enumNames:
+                        - 'None'
+                        - 'Creative Commons'
+                        - 'Apache 2'
+            dependencies:
```

Comment:
Cool use of dependencies and conditional check. I learned something new!

Reply:
Thanks! I had played around with adding new fields conditionally prior, so I tried conditionally updating existing ones and it worked!
However, we're really going to need those yaml fragments & refactor if we want to keep going with this lol